### PR TITLE
Risk effects restrictions section

### PR DESCRIPTION
**Pull request by pletale**
This PR creates a Restrictions section on the Risk Effects page, which includes:
- description of what restrictions are
- explanation of the relationship between risk exposure restrictions and risk effect restrictions 
- example risk effect restrictions table (from LBWSG model)

### CODE REVIEW

**Review by alibow — APPROVED** [review]
Looks great!

File: docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst [hunk 1]
```diff
@@ -182,3 +182,77 @@ the appropriate one in our model.
...
 
+
+Restrictions
+^^^^^^^^^^^^
+
+As with cause models, risk effects models may include restrictions, which answer
+the questions: Who does this apply to? For which population groups (e.g., age or sex group)
+is this risk effect not valid? 
+
+It is worth noting that although risk effect and risk exposure both apply to risk factors,
```
Comment (zmbc) [inline docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst:193]:
```suggestion
It is worth noting that although risk effect and risk exposure both are related to risk factors,
```

Maybe this is a bit clearer? "Apply" feels awkward to me.

File: docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst [hunk 2]
```diff
@@ -182,3 +182,77 @@ the appropriate one in our model.
...
+restrictions for these two elements function differently. Risk exposure restrictions does
```
Comment (zmbc) [inline docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst:194]:
```suggestion
restrictions for these two elements function differently. Risk exposure restrictions do
```

**Review by zmbc — APPROVED** [review]
👍

**Review by aflaxman — APPROVED** [review]

File: docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst [hunk 3]
```diff
@@ -182,3 +182,77 @@ the appropriate one in our model.
...
+    - Age group start = 2 (Early neonatal, 0-7 days)
+    - Age group end = 235 (95+)
+
+  Note that this attribution of DALYs is **not** based on the relative risks for
+  all-cause mortality, but instead is based on the logic that all preterm births
+  are due to short gestation by definition. Thus, if we include Neonatal Preterm
+  Birth in our models, the relative risks likely must be handled differently for
+  this cause.
+
+  .. todo::
```
[the remark this answers is not in the document]
Comment (pletale) [reply docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst:255]: Oh yes it did! Thanks for catching that!

**Pull request merged**